support-frontend/app/services/pricing/PriceSummaryService.scala

```diff
@@ -76,25 +77,7 @@ class PriceSummaryService(
       productRatePlan: ProductRatePlan[Product],
       price: Price,
       saving: Option[Int],
-  ) = {
-    val country = countryGroup.defaultCountry.orElse(countryGroup.countries.headOption).getOrElse(Country.UK)
-    SafeLogger.info(s"Validating promotions. Country: $country, productRatePlan: ${productRatePlan.id}")
-
-    SafeLogger.info("Promotions to validate")
-    SafeLogger.info(promotions.map(promo => promo.promoCode).mkString(", "))
-
-    val (invalidPromotions, validPromotions) = promotions
-      .map(promo =>
-        promotionService
-          .validatePromotion(promo, country, productRatePlan.id, isRenewal = false),
-      )
-      .partitionMap(identity)
-
-    SafeLogger.info("Invalid promotions")
-    SafeLogger.info(invalidPromotions.map(promoError => promoError.msg).mkString(", "))
-
-    SafeLogger.info("Valid promotions")
-    SafeLogger.info(validPromotions.map(promoError => promoError.promoCode).mkString(", "))
```

Comment:
removing these lines is the bit that makes the logging less verbose when people are hitting our page